src/Jay/Jay.fsproj

```diff
@@ -21,6 +21,7 @@
     <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <TargetFramework>netstandard2.0</TargetFramework>
+    <DisableImplicitFSharpCoreReference>true</DisableImplicitFSharpCoreReference>
```

Comment:
I added this because I was getting a 'duplicate package reference' error between the local reference to FSharp.Core 4.5.2 and the implicit reference to 7.x from the v7 SDK